lib/widgets/table.dart

```diff
@@ -167,12 +166,12 @@ class FilesTable extends StatelessWidget {
   }
 
   Widget _buildHeaderCell(FilesColumn column, int index) {
-    final double startPadding = index == 0 ? rowHorizontalPadding : 0;
+    final startPadding = index == 0 ? rowHorizontalPadding : 0;
 
     return InkWell(
       onTap: column.allowSorting
           ? () {
-              bool newAscending = ascending;
+              var newAscending = ascending;
```

Comment:
ok no, if possible remove this lint, i'd like to keep mutable variables typed where possible

Reply:
sure, thanks!

Reply:
done, and removed the lint rule